src/test/data_augmentation_test.cpp

```diff
@@ -21,6 +21,8 @@ int main( int argc, char** argv )
    data.Read("../../imgs/lena.png");
    //Apply average filter
    data.AverageFilter(kernel_size);
+   //Add salt and pepper noise
```

Comment:
Modifica los headers the todos los files para incluir tu nombre y la fecha en la que se modifico el file

Reply:
Considero que es mejor hacerlo una vez se haga merge a todos los pull request